lib/matplotlib/backends/backend_svg.py

```diff
@@ -1151,10 +1151,56 @@ def _draw_text_as_text(self, gc, x, y, s, prop, angle, ismath, mtext=None):
             weight = fm.weight_dict[prop.get_weight()]
             if weight != 400:
                 font_parts.append(f'{weight}')
+
+            def unique_iter(inp):
```

Comment:
That's actually `[*dict.fromkeys(inp)]` (which is so short that it could even be inlined with a comment).

Reply:
oh, nice!

Reply:
I replaced `unique_iter`, but didn't do anything fancier.